integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -211,6 +211,16 @@ private static void initializeGenericKubernetesApiClients() {
             apiClient //the api client
         );
 
+    clusterCrdClient =
```

Comment:
we already have clusterResClient created

Reply:
> can you resync your branch to the latest changes from cluster-reference? I see some of the code is already in cluster-reference branch.

I am in sync with cluster-reference.
[vmukkara@vmukkara-1 weblogic-kubernetes-operator]$ git pull origin cluster-reference
From https://github.com/oracle/weblogic-kubernetes-operator
 * branch            cluster-reference -> FETCH_HEAD
Already up-to-date.